action.yml

```diff
@@ -18,6 +18,12 @@ inputs:
   user-fraction:
     description: "Portion of users who should get the staged version of the app. Accepts values between 0.0 and 1.0 (exclusive-exclusive)"
     required: false
+  from-track:
+    description: "The source track to promote from. If not supplied, `beta` is used."
```

Comment:
I'm not super familiar with Actions conventions, but would it make more sense to give this a default value and set required to true, rather than hide the default in the action code?